Makefile

```
@@ -35,6 +35,7 @@ ENV_APISIX             ?= $(CURDIR)/bin/apisix
 ENV_GIT                ?= git
 ENV_TAR                ?= tar
 ENV_INSTALL            ?= install
+ENV_RM                 ?= rm
```

```suggestion
ENV_RM                 ?= rm -vf
```
let's extraction of public parts here, :)